charts/osm/templates/preset-mesh-root-certificate.yaml

```diff
@@ -41,6 +41,7 @@ data:
           "port": {{.Values.osm.vault.port | mustToJson}}
         }
         {{- end}}
-      }
+      },
+      "intent": {{.Values.osm.intent}}
```

Comment:
Thanks for adding this here! I don't think `intent` should be added as a value. If the user is using the default MRC the `intent` should be `active` (the default MRC should only be created if there are no other MRCs in the osm namespace and thus it can be assumed it is the `active` MRC). What are your thoughts on this?

Reply:
Ah gotcha, that makes sense